src/views/SpaceSettings.vue

```diff
@@ -172,6 +165,17 @@ const isViewOnly = computed(() => {
   return !(isSpaceController.value || isSpaceAdmin.value);
 });
 
+onMounted(async () => {
+  populateForm(props.space);
+  await loadEnsOwner();
+  await loadSpaceController();
+  loaded.value = true;
+
+  if (props.space.hibernated && !isViewOnly.value) {
```

Comment:
This should be moved up before await otherwise the button will show for non admins and then disappear

Reply:
Better to show it to controller with a 2 second delay rather than show it and then hide it